### PR TITLE
feat: You should be able to get an overview of what the different org roles mean without needing to view docs 

### PR DESCRIPTION
**Pull request by jayanand05**
Closes: #4243

### Description:
Get an overview of what the different org roles means without needing to view docs

https://github.com/lightdash/lightdash/assets/113882904/241e9a9a-6f4d-4f6e-aac4-92937aeaf1aa

### CODE REVIEW

File: packages/common/src/types/organizationMemberProfile.ts [hunk 1]
```diff
@@ -50,3 +50,25 @@ export type ApiOrganizationMemberProfile = {
...
+        return 'Can create, edit and delete projects';
+    }
+    if (role === OrganizationMemberRole.DEVELOPER) {
+        return 'Can create, edit and delete projects';
+    }
+    if (role === OrganizationMemberRole.ADMIN) {
+        return 'Full access to entire project, manage all projects and user access';
+    }
+    return null;
+};
```
Comment (yasmin-e) [inline packages/common/src/types/organizationMemberProfile.ts:74]: can we use a `Switch` statement here instead?

File: packages/frontend/src/components/UserSettings/UserManagementPanel/index.tsx [hunk 2]
```diff
@@ -158,6 +161,33 @@ const UserListItem: FC<{
...
+                                                display: 'block',
+                                                cursor: 'pointer',
+                                            }}
+                                        >
+                                            {label}
+                                        </Text>
+
+                                        <Tooltip multiline label={title}>
+                                            <MantineIcon
+                                                icon={IconHelp}
```
Comment (yasmin-e) [inline packages/frontend/src/components/UserSettings/UserManagementPanel/index.tsx:182]: we should add `color="gray.6"` here to match other info icons in the app

**Review by yasmin-e — CHANGES_REQUESTED** [review]
Hi @jayanand05 👋🏼 , thank you for your contribution! 
I left some comments below 👇🏼

File: packages/common/src/types/organizationMemberProfile.ts (hunk 1 above)
Comment (jayanand05) [reply packages/common/src/types/organizationMemberProfile.ts:74]: Hi @yasmin-e , yeah i will change it to switch statement

Comment (yasmin-e) [reply packages/common/src/types/organizationMemberProfile.ts:71]: thank you

**Comment by jayanand05** [comment]
hi @yasmin-e, can you explain why one of the checks failed?

**Comment by yasmin-e** [comment]
Hey @jayanand05, the check failing is just a flaky e2e test! In this case, you just have to re-run it by clicking details > re-run jobs.
Btw, I made some tiny changes (mostly code clean up, naming conventions and using Mantine <Group> component instead of a div with styling). I opened a duplicate branch [here](https://github.com/lightdash/lightdash/pull/6282) and will merge this one instead since I can't push commits to your branch!
Your commits will be included and you will appear as contributor nonetheless.
Feel free to ask me any questions here or in our [Slack community](https://lightdash-community.slack.com/ssb/redirect)!
Thank you for contributing!! ⚡

**Pull request closed without merging**